src/compute/src/sink/materialized_view.rs

```diff
@@ -697,8 +722,10 @@ mod mint {
 
             let lower = persist_frontier.clone();
             let upper = desired_frontier.clone();
-            let desc = BatchDescription::new(lower, upper);
+            let append_worker = self.next_append_worker;
+            let desc = BatchDescription::new(lower, upper, append_worker);
 
+            self.next_append_worker = (self.next_append_worker + 1) % self.worker_count;
```

Comment:
Just pointing it out, we might want to round-robin through _processes_ here. If you want, leave a comment to that note, but fine to ignore.

Reply:
Indeed. It's unclear to me how much that would change. Cycling through workers will also make us cycle through processes, just slower. Whether that makes a difference depends a lot on how persist maintenance gets assigned and I don't know enough about that to say.

Cycling through processes would also require us to know how many processes (or workers per process) there are in the cluster, and I'm not sure where to get that information. We'd have to somehow remember it from the initial `TimelyConfig`.